src/routes/transactions/mappers/multisig-transactions/multisig-transaction-execution-info.mapper.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@nestjs/common';
+import { MultisigTransaction } from '../../../../domain/safe/entities/multisig-transaction.entity';
+import { Safe } from '../../../../domain/safe/entities/safe.entity';
+import { AddressInfo } from '../../../common/entities/address-info.entity';
+import { ExecutionInfo } from '../../entities/execution-info.entity';
+import { MultisigExecutionInfo } from '../../entities/multisig-execution-info.entity';
+
+@Injectable()
+export class MultisigTransactionExecutionInfoMapper {
+  mapExecutionInfo(
+    transaction: MultisigTransaction,
+    safe: Safe,
+    txStatus: string,
```

Comment:
Why not pass the mapped `txStatus`? Then we could compare against an enum and not against a `string`

Reply:
Agreed, changed in [d985d6c](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/d985d6cb5043fb4c54881c2cdab35c7007f20306)